draft-ietf-scitt-architecture.md

```diff
@@ -428,142 +425,62 @@ Verifiers can choose which Issuers they trust.
-Each Transparency Service also defines its own Registration Policies, which MUST apply to all entries in the Append-only Log.
-
-The combination of Identity, Registration Policy evaluation, and the Transparency Service endpoint constitute the trusted part of the Transparency Service.
-Each of these components MUST be carefully protected against both external attacks and internal misbehavior by some or all of the operators of the Transparency Service.
-For instance, the code for the Registration Policy evaluation and endorsement may be protected by running in a Trusted Execution Environment (TEE).
-The Transparency Service may be replicated with a consensus algorithm, such as Practical Byzantine Fault Tolerance (pBFT {{PBFT}}) and may be used to protect against malicious or vulnerable replicas.
-Threshold signatures may be use to protect the service key, etc.
-
-Beyond the trusted components, Transparency Services may operate additional endpoints for auditing, for instance to query the history of Signed Statements registered by a given Issuer via a certain Subject.
-Implementations of Transparency Services SHOULD avoid using the service identity and extending the Transparency Service in auditing endpoints, except if it is necessary to compute an Append-only Log consistency proofs.
-Other evidence to support the correctness and completeness of the audit response MUST be computed from the Append-only Log.
+## Transparency Service {#ts-duties}
 
-### Service Identity, Remote Attestation, and Keying
+The role of a Transparency Service includes several major functions.
+The most important function is to maintain Registration Policy for the Append-only Log that is the verifiable data structure recording Signed Statements.
+All Transparency Services MUST expose APIs for Registration of Signed Statements and Receipt issuance.
+Transparency Services MAY support additional APIs for auditing, for instance to query the history of Signed Statements.
 
-Every Transparency Service MUST have a public service identity, associated with public/private key pairs for signing on behalf of the service.
-In particular, this identity must be known by Verifiers when validating a Receipt.
+### Initialization {#ts-initialization}
 
-This identity MUST be stable for the lifetime of the service, so that all Receipts remain valid and consistent.
-The Transparency Service operator MAY use a distributed identifier as their public service identity if they wish to rotate their keys, if the Append-only Log algorithm they use for their Receipt supports it.
-Other types of cryptographic identities, such as parameters for non-interactive zero-knowledge proof systems, may also be used in the future.
+The Append-only Log is empty when the Transparency Service is initialized.
+The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
+The second set of entries are Signed Statements for additional domain-specific Registration Policy.
```

Comment:
```suggestion
The second set of entries are Signed Statements for an additional domain-specific Registration Policy.
```

Reply:
There is a problem with the plurality of the sentence above. You have "Signed Statement**s**", yet you leave out "**a** Registration Policy", or Signed Statements for Registration Polic**ies**"